src/main/daml/Daml/Finance/Interface/Types/Common/Types.daml

```diff
@@ -24,22 +24,22 @@ instance Show Id where
 -- | The `HoldingStandard` data type classifies holdings based on the interfaces they implement in
 -- the holding hierarchy:
 --
---   Fungible.I
---      | require
---      v
---   Transferable.I
---      | require
---      v
---    Base.I
+-- Fungible.I    Transferable.I
+--       \  requires /
+--         v        v
+--           Base.I
 --
 data HoldingStandard
   = Fungible
-    -- ^ Represents the class of holdings which implement Fungible.I, and by extension,
-    --   Transferable.I and Base.I.
+    -- ^ Represents the class of holdings which implement Fungible.I and Transferable.I, and by
+    --   extension Base.I.
   | NonFungible
     -- ^ Represents the class of holdings which implement Transferable.I and, by extension,
     --   Base.I, but not Fungible.I.
-  | NonTransferable
+  | FungibleNonTransferable
```

Comment:
Fungible?

Reply:
Happy to discuss the proposals with the team

Reply:
changed it